src/main/java/org/pegdown/ToHtmlSerializer.java

```diff
@@ -49,7 +49,7 @@ public ToHtmlSerializer(LinkRenderer linkRenderer) {
     }
 
     public ToHtmlSerializer(LinkRenderer linkRenderer, List<ToHtmlSerializerPlugin> plugins) {
-    	this (linkRenderer, Collections.<String, VerbatimSerializer>emptyMap(), plugins);
```

Comment:
I can’t live without auto-formatting. I just type the code and hit reformat, I don’t waste time on fixing the format. It is hard when I cannot do it.

If you have a chance, you may want to do just a re-format commit without any changes so that the code is consistently formatted. I find it easier to follow. I don’t really care what the formatting style is, just as long as it is consistent.


Best regards,


Vladimir.

> On Aug 22, 2015, at 5:37 PM, Mathias <notifications@github.com> wrote:
> 
> In src/main/java/org/pegdown/ToHtmlSerializer.java <https://github.com/sirthias/pegdown/pull/181#discussion_r37700032>:
> 
> > @@ -49,7 +49,7 @@ public ToHtmlSerializer(LinkRenderer linkRenderer) {
> >      }
> >
> >      public ToHtmlSerializer(LinkRenderer linkRenderer, List<ToHtmlSerializerPlugin> plugins) {
> > -    	this (linkRenderer, Collections.<String, VerbatimSerializer>emptyMap(), plugins);
> Nice formatting catch!
> One reason why I have switched to auto-formatting for all current projects in the mean-time...
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sirthias/pegdown/pull/181/files#r37700032>.
>